Makefile

```diff
@@ -17,6 +17,12 @@ build: go.sum
 test:
 	$(GO) test -v ./...
 
+lint:
+	GO111MODULE=off go get github.com/golangci/golangci-lint/cmd/golangci-lint
+	golangci-lint run --timeout 5m0s --allow-parallel-runners
+	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" | xargs gofmt -d -s
+	go mod verify
```

Comment:
This won't used by CI, but for running this on your local?

Reply:
Yes, It used only local.